beta/project-griffon/set-up-project-griffon.md

```diff
@@ -140,8 +150,11 @@ public static void startSession(final String url)
  Griffon.startSession(url);
 ```
 {% endtab %}
-
 {% tab title="iOS" %}
+
+Once your app is configured to handle the deeplink. You can start the griffon session by opening the deeplink provided from the griffon UI
```

Comment:
Please reword as follows:
"After your app is configured to handle the deeplink, start the Griffon session by opening the deeplink from the Griffon UI."